docs/sources/release-notes/release-notes-8-1-0-beta2.md

```diff
@@ -0,0 +1,22 @@
++++
+title = "Release notes for Grafana 8.1.0-beta2"
+[_build]
+list = false
++++
+
+<!-- Auto generated by update changelog github action -->
+
+# Release notes for Grafana 8.1.0-beta2
+
+### Features and enhancements
+
+* **Alerting:** Expand the value string in alert annotations and labels. [#37051](https://github.com/grafana/grafana/pull/37051), [@gerobinson](https://github.com/gerobinson)
+* **Auth:** Add Azure HTTP authentication middleware. [#36932](https://github.com/grafana/grafana/pull/36932), [@kostrse](https://github.com/kostrse)
+* **Auth:** Pass user role to Grafana using auth proxy. [#36729](https://github.com/grafana/grafana/pull/36729), [@yuwaMSFT2](https://github.com/yuwaMSFT2)
+* **Gazetteer:** Update Countries JSON. [#37129](https://github.com/grafana/grafana/pull/37129), [@bryanuribe](https://github.com/bryanuribe)
+
+### Bug fixes
+
+* **Config:** Fix Docker builds by removing bad formatting in sample.ini. [#37106](https://github.com/grafana/grafana/pull/37106), [@FZambia](https://github.com/FZambia)
```

Comment:
What does "bad" mean here? Typically, we avoid judgement terminology because it does not describe the actual problem. Would my suggestion work?

```suggestion
* **Config:** Fix Docker builds by correcting formatting in sample.ini. [#37106](https://github.com/grafana/grafana/pull/37106), [@FZambia](https://github.com/FZambia)
```